trust-quorum/src/coordinator_state.rs

```diff
@@ -86,6 +90,7 @@ impl CoordinatorState {
 
     /// A reconfiguration from one group to another
     pub fn new_reconfiguration(
+        log: Logger,
```

Comment:
Same q here -- add `component` context?

Reply:
Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)